metabrainz/templates/index/privacy.html

```diff
@@ -7,12 +7,144 @@ <h1 class="page-title">{{ _('Privacy Policy') }}</h1>
   <p>
-    {{ _('The MetaBrainz Foundation will not collect any private information unless
-    you choose to donate to support the MusicBrainz project. Any personal
-    information you choose to provide during the donation process will not be
-    revealed to anyone else, unless you give us permission to list your name on
-    our donations page. We will not contact you unless you give us permission
-    to do so when you make a donation.') }}
+  <ol>
+    <li>
+      You do not have to provide any personal data to be able to browse the contents of the databases published by MetaBrainz.
+    </li>
+    <li>
+      You do not have to provide any personally-identifying information if you choose not to.
+    </li>
+    <li>
+      We will never reveal your email address on MetaBrainz controlled web sites.
+    </li>
+  </ol>
+  <p>
+    In order to get maximum value out of the MetaBrainz projects, you may want to create an account and log in. 
+    Doing so grants you the ability to: edit the databases, contribute your own data, communicate with other users, and keep track 
+    of recently released music from artists in your MusicBrainz collection. If you choose to create an account then the minimum we 
+    ask is that you: choose a unique username and password, use a web browser that accepts "session" cookies, and provide a verified 
+    email address.
+  </p>
+  <p>
+    If you do not create an account (or are not logged in) you will not have access to the above listed features, however, you will 
+    still have full access to browse and examine the database.
+  </p>
+
+  <h2>{{ _('Applicable to all users') }}</h2>
+
+  <h3>{{ _('Cookies') }}</h3>
+  <p>
+    We use two cookies that contain sensitive information:
+  </p>
+
+  <ol>
+    <li>
+      <b>remember_login</b>: If you choose the "Log in permanently" option when logging in to one of our sites; the cookie will be used to 
+      remember your login details.
+    </li>
+    <li>
+      <b>musicbrainz_server_session</b>: If you are logged into one of our sites, your current session ID is stored in this cookie.
+    </li>
+  </ol>
+
+  <p>
+    We may store boolean dynamic preference information in other cookies, but these do not contain any sensitive information.
+  </p>
+
+  <h3>{{ _('Web and FTP access logs') }}</h3>
+
+  <p>
+    In a practice similar to other web sites we keep logs of all web requests made against our servers. These logs include: your IP 
+    address, your browser's "User-Agent" string, and which page you requested. Aggregate information about web and FTP traffic is 
+    made available to the public via our site usage pages.
+  </p>
+
+  <h3>{{ _('Third-Party content') }}</h3>
+  <p>
+    Our project's web pages may load some third-party content. Some, but possibly not all sites are listed below:
+  </p>
+
+  <ol>
+    <li>
+      Cover art images: These are provided by the various cover art sites that have given MusicBrainz permission to do so. This means 
```

Comment:
Maybe give CAA a special mention here? And maybe generalise a bit to also talk about artist (and label and place and area?) images that are pulled from WM Commons?

(But this could also reasonably be for another ticket/PR and not included in this one. This bullet point is still correct without any of these changes.)

Reply:
I've made some minor changes here that should make this better.

Reply:
Changes here LGTM.